incydr/cli/render.py

```diff
@@ -19,24 +34,38 @@ def date_time(dt: datetime):
     return dt.isoformat(sep=" ")[:16] + " UTC"
 
 
-def model_as_row(model: BaseModel, columns: Set[str] = None):
-    for name, value in model:
-        if columns and name not in columns:
-            continue
-        if isinstance(value, datetime):
-            yield date_time(value)
-        else:
-            yield str(value)
+def table(
+    model: Type[BaseModel],
+    models: Iterable[BaseModel],
+    columns: list[str] = None,
+    title=None,
+    flat=False,
+):
+    headers = list(get_fields(model, include=columns, flat=flat))
+    header_padding = len(headers) * 3
+    max_width = 0
+    tbl = Table(*headers, title=title, show_lines=True)
+    for m in models:
```

Comment:
Do both of these methods handle no results/empty generators?

Reply:
Good call, they don't... yet.